package.json

```diff
@@ -22,6 +22,7 @@
     "prop-types": "~15.6.0",
     "react": "~16.8.0",
     "react-dom": "~16.8.0",
+    "react-dropzone": "10.0.0",
```

Comment:
Since you're not implementing this yet, please `yarn remove react-dropzone`

Reply:
Done!